api/data_explorer/controllers/export_url_controller.py

```diff
@@ -166,13 +190,24 @@ def _get_sql_query(filters):
         facet_id = splits[0]
         value = splits[1]
         field_type = ''
+        is_time_series = False
```

Comment:
Seems like is_time_series and bucket_interval will always be defined in one of the if statements below. So not necessary to declare these here.